src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.iOS.cs

```diff
@@ -71,6 +71,14 @@ public override bool BecomeFirstResponder()
 			_textBoxView?.UpdateTextAlignment();
 		}
 
+		partial void SelectPartial(int start, int length)
+		{
+			if (_textBoxView != null)
+			{
+				_textBoxView.SelectedTextRange = _textBoxView.GetTextRange(start: start, end: start + length);
```

Comment:
Currently for macOS, this equivalent code path goes through a new method in `ITextBoxView`, but for iOS I just use `_textBoxView.SelectedTextRange` directly.

Should iOS work the same way as macOS? I thought this way is simpler (but can't do macOS since `SelectedRange` in `INSTextInput` is readonly. Does it actually matter?